src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -13,7 +13,17 @@
 
 namespace Sonata\MediaBundle\DependencyInjection;
 
+use Sonata\ClassificationBundle\Model\CategoryInterface;
 use Sonata\EasyExtendsBundle\Mapper\DoctrineCollector;
+use Sonata\MediaBundle\CDN\CloudFront;
```

Comment:
Is this use really used? Because you removed the function that would need it.

I am not sure why travis didn't complain about it

Reply:
Uuuh yeah that's weird php-cs-fixer doesn't remove it.